native/src/modules/endpoint/sagas/loadCityContent.js

```diff
@@ -68,40 +149,9 @@ export default function* loadCityContent(
     }
 
     if (criterion.shouldLoadLanguages()) {
-      try {
-        yield call(
-          loadLanguages,
-          newCity,
-          dataContainer,
-          shouldUpdate
-        ) /* The languages for a city get updated if a any
-                                                                       language of the city is:
-                                                                          * older than MAX_CONTENT_AGE,
-                                                                          * has no "lastUpdate" or
-                                                                          * an update is forced
-                                                                        This means the loading of languages depends on
-                                                                        language AND the city */
```

Comment:
This comment should stay I think

Reply:
I think it is self explanatory and the same for all data fetched from the endpoints with an exception from languages. But I can also readd it.